tueplots/axes.py

```diff
@@ -3,14 +3,14 @@
 
 def lines(
     *,
-    base_width=1.0,
+    base_width=0.75,
     color="black",
     spines_left=True,
     spines_right=True,
     spines_top=True,
     spines_bottom=True,
-    xtick_direction="out",
-    ytick_direction="out",
+    xtick_direction="inout",
+    ytick_direction="inout",
 ):
```

Comment:
now that I looked at the plonts, these seem like much better defaults. (But this Is very subjective.)